### PR TITLE
KAA-1242 Programming guide -> Using Kaa endpoint SDKs -> Supported pl…

### PR DESCRIPTION
**Pull request by balldir**
…atform matrix
NOTE: KAA-1157 was updated with request to update design for tables
NOTE: KAA-1253 was created to investigate platform support graduation
Proposed changes  are heavily using Liquid Templating Engine to increase readability.

### CODE REVIEW

**Comment by Slash32** [comment]
Looks good to me

**Comment by forGGe** [comment]
+

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/index.md [hunk 1]
```diff
@@ -4,4 +4,49 @@ title: Supported platforms
...
+{% capture LCPP %}{{sdk_cpp}}/SDK-Linux/{% endcapture %}
+{% capture RCPP %}{{sdk_cpp}}/SDK-RPi/{% endcapture %}
+{% capture BCPP %}{{sdk_cpp}}/SDK-Beaglebone/{% endcapture %}
+{% capture SCPP %}{{sdk_cpp}}/SDK-Samsung-Artik-5/{% endcapture %}
+{% capture ICPP %}{{sdk_cpp}}/SDK-Edison/{% endcapture %}
+
+{% capture sdk_obj %}{{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/Objective-C{% endcapture %}
+{% capture IO %}{{sdk_obj}}/SDK-iOS/{% endcapture %}
+
+{% capture sdk_j %}{{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/Java/{% endcapture %}
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/index.md]: s/sdk_j/sdk_java

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/index.md [hunk 2]
```diff
@@ -4,4 +4,49 @@ title: Supported platforms
...
+{% capture AJ %}{{sdk_j}}{% endcapture %}
+{% capture DJ %}{{sdk_j}}{% endcapture %}
+{% capture LJ %}{{sdk_j}}{% endcapture %}
+{% capture X %}Supported{% endcapture %}
+
+{: id="supportedPlatformsTable"}
+|Platform                |[C]({{sdk_c}}) |[C++]({{sdk_cpp}})|[Objective-C]({{sdk_obj}})|[Java]({{sdk_j}})| 
+| -------                | ---           | ---              | ---                      | ---             |
+|Linux                   |[{{X}}]({{LC}})|[{{X}}]({{LCPP}}) |                          |[{{X}}]({{LJ}})  |
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/index.md]: Add spaces around `|`?

**Comment by rasendubi** [comment]
LGTM

**Pull request merged**